docs/build-the-docs.md

```diff
@@ -467,6 +468,15 @@ Follow this procedure to show and use search and populate an index:
     yarn serve
     ```
 
+## Resolving Edit-this-page in Development
+
+When doing a local build, the link `Edit this page` on the top right resolves to the local file system. If you want to make Antora to resolve to where the files are sourced on github, you need to build using the envvar `CI=true`. This can be done like with the following command:
```

Comment:
```suggestion
When doing a local build, the link `Edit this page` on the top right resolves to the local file system. If you want to make Antora to resolve to where the files are sourced on github, you need to build using the envvar `CI=true`. This can be done with the following command:
```